media/CircleGraph/view.js

```diff
@@ -542,13 +542,24 @@ view.View = class {
     }
   }
 
+  /**
+   * @brief called when it's needed to change scroll behaviour
+   */
+  setScrollToSelected(value) {
+    this._scrollToSelected = value;
+  }
+
   /**
    * @brief toggleSelect will select or toggle select with CtrlKey down
    * @param viewNode view.Node instance
    * @note  works on host mode is viewMode.selector
    */
   toggleSelect(viewNode) {
-    if (viewNode && this._host._mode === viewMode.selector) {
+    if (
```

Comment:
How about returning 'if not' condition here to reduce the level of condition?
Like,
```
if (!viewMode) {return}
if (this._host._mode === viewMode.selector || this._host._mode === viewMode.visqselector) {return}

// continued logic
```

Reply:
@dayo09 
Thank you. Unfortunately there are 4 values, including `visqselector`, so inverting logic will still produce:
```
if (this._host._mode !== viewMode.viewer && this._host._mode !== viewMode.visq) {return}

// continuous logic
```

Reply:
I mean, if it's not 'viewMode', then this function don't need to proceed. Note that it's an 'and' condition. 😄

Reply:
Ahhh. Sorry. I didn't immediately understand that.

Reply:
@dayo09 
fixed.